### DISCUSSION TITLE
Is it possible to remove classes gx-lg-4 gx-xl-5 gx-xxl-5 from astroid-row?

### DISCUSSION THREAD

**Discussion by boldc** [templaza/astroid-framework#635] [General]
The Astroid templates' output includes the following classes in rows: **astroid-row row gx-lg-4 gx-xl-5 gx-xxl-5**

The classes **gx-lg-4 gx-xl-5 gx-xxl-5** are creating a lot of empty space because the left/right padding is 3rem.

Is there an easy way to prevent Astroid from applying these gx classes to rows? Or an easy wat to change the gx CSS values?

Using SASS override for **grid-gutter-width** does not resolve the issue. Any help is greatly appreciated

![remove-css-classes](https://github.com/templaza/astroid-framework/assets/15841527/159586a9-6cdb-4e60-a885-14e2e0798e64)

**Comment by Chacapamac** [comment]
I just put the Bootstrap reference for (gx-) Gutters, if needed —> https://getbootstrap.com/docs/5.0/layout/gutters/

I just check in a clean Astroid install and the only (gx-) reference I find is (gx-0)

Did you use a widget or maybe this code is added to the custom class setting on the specific Astroid Layout position?

**Reply by boldc** [reply]
I couldn't find any widget using this CSS class, and I didn't add any CSS classes to layout positions. This is for Astroid Template Two, by the way. I did not see this markup when I switched to Template Zero and Template One.

**Comment by sonvnn** [comment]
@boldc It's very simple. 
1. Just go to Layout tab -> Layout Configuration -> Deselect "Extra Extra Large" and "Extra Large" https://docs.astroidframe.work/layout-settings/layout-configuration-on-devices
2. Go to Row settings -> Switch "Extra Large Gutter" and "Extra Extra Large Gutter" to Inherit
![Screen Shot 2024-04-08 at 09 47 07](https://github.com/templaza/astroid-framework/assets/3204103/f65676d6-9249-4b78-8e40-7b470c91c01d)
![Screen Shot 2024-04-08 at 09 44 39](https://github.com/templaza/astroid-framework/assets/3204103/633591a6-a259-4396-a5cd-60babe21c310)

**Reply by boldc** [reply]
Perfect. thank you @sonvnn!
